src/ort_genai.h

```diff
@@ -236,9 +240,61 @@ struct OgaGenerator : OgaAbstract {
   static void operator delete(void* p) { OgaDestroyGenerator(reinterpret_cast<OgaGenerator*>(p)); }
 };
 
+struct OgaTensor : OgaAbstract {
+#if __cplusplus >= 202002L
+  static std::unique_ptr<OgaTensor> Create(void* data, std::span<const int64_t> shape, OgaElementType element_type) {
+    OgaTensor* p;
+    OgaCheckResult(OgaCreateTensorFromBuffer(data, shape.data(), shape.size(), element_type, &p));
+    return std::unique_ptr<OgaTensor>(p);
```

Comment:
This is a wrapper around the C API, so it can't call make_unique as there's no C++ object to be constructed. We just get back the pointer from the C API.